src/game.h

```diff
@@ -317,6 +317,7 @@ class Game
 		void playerReportBug(uint32_t playerId, const std::string& message, const Position& position, uint8_t category);
 		void playerDebugAssert(uint32_t playerId, const std::string& assertLine, const std::string& date, const std::string& description, const std::string& comment);
 		void playerAnswerModalWindow(uint32_t playerId, uint32_t modalWindowId, uint8_t button, uint8_t choice);
+		void playerReportRuleViolationReport(unsigned playerid, std::string target_name, RuleViolationType_t report_type, RuleViolationReasons_t report_reason, const std::string player_comment, const std::string translation);
```

Comment:
Strings should be passed as constant reference. `std::string target_name` -> `const std::string& targetName`
Use fixed width integers. `unsigned playerid` -> `uint32_t playerId`
Use Camel casing for variables (as you can see in my examples).